Feature - upload files to specific folder on Azure File Share (#112)
arcus-azure/arcus.scripting: docs/preview/features/powershell/azure-storage-fileshare.md

**Example**

```powershell
PS> Create-AzFileShareStorageFolder -ResourceGroupName "shipping-resources" -StorageAccountName "tracking-account-storage" -FileShareName "returned" -FolderName "containers"
# Creating 'containers' directory in file share..
# Directory 'containers' has been created..
```

## Copying files to a folder on an Azure file share

> Upload a series of files at a given folder, matching an optional file mask, to a Azure File Share.

Reviewer: ```suggestion
Upload a set of files from a given folder, optionally matching a specific file mask, to an Azure File Share.
```

| Parameter               | Mandatory | Description                                                                                                            |
| ----------------------- | --------- | ---------------------------------------------------------------------------------------------------------------------- |
| `ResourceGroupName`     | yes       | The resource group containing the Azure File Share.                                                                    |

> | `StorageAccountName`    | yes       | The Azure Storage account name that is hosting the Azure File Share.                                                   |

Reviewer: ```suggestion
| `StorageAccountName`    | yes       | The name of the Azure Storage account that is hosting the Azure File Share.                                                   |
```
